packages/flutter_tools/templates/app/web/index.html.tmpl

```diff
@@ -15,6 +15,16 @@
   <link rel="manifest" href="/manifest.json">
 </head>
 <body>
+  <!-- This script installs service_worker.js to provide PWA functionality to
+       application. For more information, see:
+       https://developers.google.com/web/fundamentals/primers/service-workers -->
+  <script>
+    if ('serviceWorker' in navigator) {
+      window.addEventListener('load', function () {
+        navigator.serviceWorker.register('/service_worker.js');
```

Comment:
:bug: here - other places are calling this `flutter_service_worker.js` - this template produces an error message:

> Uncaught (in promise) TypeError: Failed to register a ServiceWorker for scope ('http://localhost:8000/') with script ('http://localhost:8000/service_worker.js'): A bad HTTP response code (404) was received when fetching the script.

Reply:
Ahh, forgot to update this one. Thanks for the heads up!

Reply:
Fix pending here: https://github.com/flutter/flutter/pull/48778

Reply:
Hello. I have same errror. Created issue: https://github.com/flutter/flutter/issues/49479